horizons/ai/aiplayer/mission/specialdomestictrade.py

```diff
@@ -114,8 +118,12 @@ def _reached_source_settlement(self):
 			self.report_failure('No resources to transport')
 
 	def _move_to_destination_settlement(self):
-		self._move_to_warehouse_area(self.destination_settlement_manager.settlement.warehouse.position, Callback(self._reached_destination_settlement),
```

Comment:
Let my explain my change: I make a new line after ( and , with | mark a new line:
```
		self._move_to_warehouse_area(self.source_settlement_manager.settlement.warehouse.position, Callback(self._reached_source_settlement),
                                             |                                                            |                                          |
			Callback(self._move_to_source_settlement), 'Unable to move to the source settlement (%s)' % self.source_settlement_manager.settlement.get_component(NamedComponent).name)
                                                                  |                                                |
```
Is it possible to accept this in this commit?

This construct is in
https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L94-L99

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L128-L133

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L173-L178

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/specialdomestictrade.py#L79-L84

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/specialdomestictrade.py#L121-L126